ui/raidboss/data/05-shb/trial/varis-ex.txt

```diff
@@ -0,0 +1,116 @@
+
+# Phase 2
+200.1 "--targetable--"
+233.2 "Vivere Militare Est" sync /:Bladesblood:4CCF:/ duration 30 window 80,50
+
+# Phase 3 (random groups)
+262.3 "Gunshield Actions" # fake action here for placeholding
+
+
+# ================= random groups ===================
+# group 1 (Loaded Gunshield, spread)
+300.0 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/ window 100,400
+308.9 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+313.2 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+314.7 "Terminus Est"
+316.3 "Magitek Burst" sync /:Varis Yae Galvus:4CDF:/
+317.0 "Ignis Est Action"
+324.3 "Terminus Est" # sync /:Terminus Est:4CB4:/
+324.5 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+# since trigger would notice tankbuster with startsUsing, timeline would not show it
+# 312.7 "Citius" sync /:Varis Yae Galvus:4CF0:/
+
+
+# group 2 (Reinforced Gunshield, block)
+500.0 "Reinforced Gunshield" sync /:Varis Yae Galvus:4CD9:/ window 300,300
+509.1 "Altius" sync /:Varis Yae Galvus:4CCA:/
+518.0 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+519.2 "Terminus Est" # sync /:Terminus Est:4CB4:/
+521.3 "Magitek Shielding" sync /:Varis Yae Galvus:4CDB:/
+526.1 "Ventus Est Action" sync /:Ventus Est:4CC7:/
+
+# group 3 (Electrified Gunshield, knockback)
+700.0 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/ window 500,100
+705.1 "Reinforcements" sync /:Varis Yae Galvus:4CEA:/
+711.0 "Terminus Est" # sync /:Phantom Varis:4CB3:/
+714.2 "Aetherochemical Grenado" sync /:Magitek Turret II:4CED:/
+723.9 "Terminus Est" # sync /:Terminus Est:4CB4:/
+726.6 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+737.5 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+# ==================== random groups ==================
+
+
+# Phase 4 
+1000.0 "--sync--" sync /:Varis Yae Galvus:4CE0:/ window 800,20
+1010.4 "Magitek Spark/Torch" sync /:Gunshield:(4CE4|4CE3):/ window 10,5
+1019.1 "Magitek Torch/Spark" sync /:Gunshield:(4CE4|4CE3):/ window 5,5
+
+# Phase 5
+1056.8 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/ window 56.8,5
+1066.0 "Fortius" 
+1078.5 "Magitek Burst" sync /:Varis Yae Galvus:4CDE:/
+1086.7 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+1097.9 "Citius" sync /:Varis Yae Galvus:4CF0:/
+
+1106.7 "Terminus Est"
+1111.1 "Electrified Gunshield" sync /:Varis Yae Galvus:4CD7:/
+1120.3 "Fortius" 
+1137.7 "Magitek Shock" sync /:Varis Yae Galvus:4CDA:/
+1151.5 "Terminus Est"
+1155.9 "Altius" sync /:Varis Yae Galvus:4CCA:/
+1164.3 "Festina Lente" sync /:Varis Yae Galvus:4CC9:/
+1171.2 "Ventus Est Ready" sync /:Varis Yae Galvus:4CC6:/
+1178.7 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+1184.4 "Terminus Est"
+1184.4 "Ventus Est Action" sync /:Ventus Est:4CC7:/
+1195.7 "Citius" sync /:Varis Yae Galvus:4CF0:/
+
+1205.4 "Ignis Est Ready" sync /:Varis Yae Galvus:4CB5:/
+1213.1 "Alea Iacta Est" sync /:Varis Yae Galvus:4CD2:/
+1218.6 "Ignis Est Action"
+
+1230.1 "Loaded Gunshield" sync /:Varis Yae Galvus:4CD8:/
```

Comment:
Sorry, one last thing. `test_timeline` seems to be telling me that this timestamp is early. It should be 1236.6, or close to that.

Reply:
yes, this part was created according to my friend's cheat sheet, I have no more logs file record till that time, my party always wipe before it.

Reply:
when I review my friend's log, this time was like 1235.69, I would change it as that one.